index.md

```diff
@@ -18,6 +18,7 @@ censorship-resistant global "social" network** once and for all.
 - It doesn't rely on any trusted central server, hence it is resilient.
 - It is based on cryptographic keys and signatures, so it is tamperproof.
 - It does not rely on P2P techniques, therefore it works.
+- Howto Start with Nostr [Tutorial in Italian](https://www.youtube.com/watch?v=m0yhG-1vNt4)
```

Comment:
it is a "how to start" video in italian. it fits good in your resource list. But if the question is to create a new section, yes why not. in my opinion also the existing list where i put may fit, because it is a video to start moving first steps. But if you prefer to create a different area, let's decide how to do it